src/styles/components/_jca-credit-card.scss

```diff
@@ -0,0 +1,192 @@
+.jca-credit-card {
+  border-radius: $default_radius;
+  border-radius: 8px;
+  background: $black;
+  overflow: hidden;
+  box-shadow: $default_shadow;
+  margin-bottom: 20px;
+  max-width: 550px;
+
+  &__logo {
+    padding: $light_padding;
+    text-align: right;
+    max-height: 80px;
+    height: 80px;
+
+    img {
+      height: 90%;
+    }
+  }
+
+  &__card-number {
+    padding: $default_padding;
+    color: $white;
+    font-size: 2em;
+    letter-spacing: 2px;
+    text-align: center;
+
+    input {
+      width: 100%;
+      background: none;
+      border: none;
+      color: $white;
+      text-align: center;
+      letter-spacing: 2px;
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &__meta {
+    display: flex;
+    color: $white;
+    padding: $default_padding;
+    padding-top: 0;
+  }
+
+  &__owner {
+    flex: 3;
+
+    input {
+      width: 100%;
+      background: none;
+      border: none;
+      color: $white;
+      text-transform: uppercase;
+      padding: 0;
+      margin: 0;
+      font-weight: 700;
+    }
+  }
+  
+  &__expiry {
+    flex: 1;
+    text-align: right;
+  }
+
+  &__month {
+    display: inline-block;
+    border: none;
+    width: 30px;
+    text-align: right;
+    background: none;
+    color: $white;
+    font-weight: 300;
+  }
+
+  &__year {
+    display: inline-block;
+    border: none;
+    width: 40px;
+    text-align: left;
+    background: none;
+    color: $white;
+    font-weight: 700;
+  }
+
+  &__cvv {
+    flex: 1;
+    display: inline-block;
+    text-align: right;
+  }
+
+  &__cvv-digits {
+    display: inline-block;
+    border: none;
+    width: 30px;
+    text-align: right;
+    background: none;
+    color: $white;
+    font-weight: 700;
+  }
+
+  &__label {
+    text-transform: uppercase;
+    font-size: 0.8em;
+  }
+
+  &__text {
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+
+  &__type {
+    font-weight: 300;
+    color: $white;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    font-size: 0.9em;
+  }
+}
+
+// GRADIENTS
+.jca-credit-card {
+  &--gradient-1 {
+    background: linear-gradient(123deg, #FFFFFF 0%, #00B2FF 100%),
+                linear-gradient(236deg, #BAFF99 0%, #005E64 100%),
+                linear-gradient(180deg, #FFFFFF 0%, #002A5A 100%),
```

Comment:
Codacy found an issue: [Expected "#FFFFFF" to be "#ffffff" (color-hex-case)](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5715280)